expfmt/openmetrics_create_test.go

```diff
@@ -20,20 +20,15 @@ import (
 	"testing"
 	"time"
 
-	"github.com/golang/protobuf/proto"  //nolint:staticcheck // Ignore SA1019. Need to keep deprecated package for compatibility.
-	"github.com/golang/protobuf/ptypes" //nolint:staticcheck // Ignore SA1019. Need to keep deprecated package for compatibility.
-
 	dto "github.com/prometheus/client_model/go"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Import nit as before: Have the named imports in their own block after the unnamed imports (as it was before).